src/store/user.js

```diff
@@ -55,18 +58,25 @@ const actions = {
   sendToken({ commit, dispatch }, userdata) {
     return new Promise((resolve, reject) => {
       login(userdata)
-        .then(response => {
+        .then((response) => {
           const { data } = response
           commit('SET_TOKEN', userdata.idToken)
           setToken(userdata.idToken)
-          setExpiredTime(Date.now() + 3 * 24 * 60 * 60 * 1000)
+          setExpiredTime(Date.now() + 30 * 60 * 1000)
           resolve(data.message)
         })
-        .catch(error => {
+        .catch((error) => {
           reject(error)
         })
     })
   },
+  async refreshToken({ commit }) {
```

Comment:
the function doesn't return a promise, but is invoked in `src/store/articles.js`, which may not know the error happened

Reply:
removed. call 'getToken' instead.